crypto/rand/rand_lcl.h

```diff
@@ -128,7 +128,12 @@ extern RAND_BYTES_BUFFER rand_bytes;
 extern RAND_DRBG rand_drbg;
 extern RAND_DRBG priv_drbg;
 
-/* How often we've forked (only incremented in child). */
+/*
+ * A "generation count" of forks.  Increment-only, and only ever written to
+ * in the child process of the fork, which is guaranteed to be single-threaded.
+ * As such, no locking is needed for normal (read) accesses, since the rest of
+ * fork processing is assumed to introduce the necessary memory barriers.
```

Comment:
Incorrect -- it's not the fork/syscall processing it's the pthreads implementation that handles it.

Reply:
How about simply writing "pthread fork processing" instead of "fork processing"?

Reply:
Sounds good to me